app_spec/test/test.js

```diff
@@ -1,28 +1,47 @@
 const test = require('tape');
-const Container = require('../../nodejs_container');
-const {pollFor} = require('./util');
+const { pollFor } = require('./util');
 
-const app = Container.instanceFromNameAndDna("bob", "dist/app_spec.hcpkg")
-app.start()
+const { ConfigBuilder, Habitat } = require('../../nodejs_container');
 
-const app2 = Container.instanceFromNameAndDna("alice", "dist/app_spec.hcpkg")
-app2.start()
+const agentAlice = ConfigBuilder.agent("alice");
+const agentBob = ConfigBuilder.agent("bob");
+
+const dnaPath = "./dist/app_spec.hcpkg";
+const dna = ConfigBuilder.dna(dnaPath);
+
+const instanceAlice = ConfigBuilder.instance(agentAlice, dna);
+const instanceBob = ConfigBuilder.instance(agentBob, dna);
+
+const config = ConfigBuilder.habitat(instanceAlice, instanceBob);
+
+const hab = new Habitat(config);
+hab.start();
```

Comment:
Please take a good look at the config building up to this point, this is likely how JS test files will all begin. Let's get this syntax right.